torch/optim/lr_scheduler.py

```diff
@@ -226,7 +226,11 @@ class CosineAnnealingLR(_LRScheduler):
 
     .. math::
         \eta_{t+1} = \eta_{min} + (\eta_t - \eta_{min})\frac{1 +
-        \cos(\frac{T_{cur+1}}{T_{max}}\pi)}{1 + \cos(\frac{T_{cur}}{T_{max}}\pi)}
+        \cos(\frac{T_{cur+1}}{T_{max}}\pi)}{1 + \cos(\frac{T_{cur}}{T_{max}}\pi)},
+        T_{cur} \neq (2k+1)T_{max};\\
```

Comment:
Verified this condition. It's so confusing that we use a different `cur+1` convention in the code and math lol.